package.json

```
@@ -22,7 +22,7 @@
   ],
   "homepage": "http://tldr-pages.github.io",
   "engines": {
-    "node": ">=6.12.0"
+    "node": ">=12.0.0"
```

```suggestion
    "node": ">=12"
```
You don't need to fully specify a version.